package.json

```diff
@@ -80,6 +80,7 @@
 	},
 	"ava": {
 		"workerThreads": false,
+		"concurrency": 1,
```

Comment:
Unfortunately, it seems like this needs to be brought back. 
Otherwise, Windows tests still seem to be randomly failing in CI: https://github.com/sindresorhus/execa/actions/runs/7878070501/job/21495511609